README.md

```diff
@@ -70,6 +70,30 @@ Then configure the rules you want to use within `rules` property of your `.eslin
 }
 ```
 
+### Run the plugin only against test files
+
+With the default setup mentioned before `eslint-plugin-testing-library` will be run against your whole codebase. If want to run this plugin only against your tests files, you can do that by using [ESLint `overrides`](https://eslint.org/docs/user-guide/configuring/configuration-files#configuration-based-on-glob-patterns).
+
+Assuming you are using the same pattern for your test files as [Jest by default](https://jestjs.io/docs/configuration#testmatch-arraystring), the following config would run `eslint-plugin-testing-library` only against your test files:
+
+```javascript
+// .eslintrc
+{
+  // here you have your usual config which applies to the whole project...
+  extends: ['airbnb', 'plugin:prettier/recommended'],
+  plugins: ['react-hooks'],
+
+  overrides: [
+    {
+      // ... and here you expand the ESLint plugins run for your tests files
+      files: ['**/__tests__/**/*.[jt]s?(x)', '**/?(*.)+(spec|test).[jt]s?(x)'],
+      extends: ['plugin:testing-library/react'],
+      plugins: ['testing-library'],
```

Comment:
No need to do this, as this is already included in the config I think?

Reply:
Good point, I'll move this to the main config side.